### DISCUSSION TITLE
FA not present in AA

### DISCUSSION THREAD

**Discussion by piotrklementowski** [AndreyPavlenko/Fermata#268] [General]
I am losing my hear here, spent 2 days and nothing. Fermata auto was working and next day it is not present in AA. My current setup
* pixel 6pro
*  Android 13 TQ2A.230505.002.G1 (can't be rooted, verizon model)
* AA 8.5.625214-release (installed manually because newest 9.6 doesn't show FA as well)
* Fermata Auto 1.9.0 (doensn't work on 1.9.2 as well) installed through kingInstaller 1.4

When I turn on AA dev mode and allow unknown source only Fermata Control is present. 
I know the problem was discussed before with exactly the same symptoms but I tried many things and nothing helps.

**Comment by fcaronte** [comment]
Can you try to install this package installer version and then try again to reinstall from scratch fermata auto using kinginstaller?

https://www.apkmirror.com/apk/google-inc/package-installer/package-installer-13-8768315-release/package-installer-13-8768315-android-apk-download/

**Reply by piotrklementowski** [reply]
worked, thanks!

**Reply by fcaronte** [reply]
Seem you can use and reinstall the same apk version you just have

**Reply by collectablessports** [reply]
You are a lifesaver! Worked for me too!

**Reply by faifer13** [reply]
Hi, this solution works for pixel 6 pro? Can after upgrade the package installer?

**Reply by fcaronte** [reply]
Yes will work, and at the end you can simply reinstall your same package installer version

**Reply by LasGeo64** [reply]
Yes! It really works on my pixel 6 pro. Happy~

**Reply by SotYPL** [reply]
Worked for my P7 on latest Android 13. Thank you!

**Reply by LasGeo64** [reply]
@fcaronte Dear fcaronte, do you have any idea that the issue will happen again in android14?
I consider to upgrade to 14 but not sure if the same installer patch can work or there is new patch for android 14.

**Reply by fcaronte** [reply]
> @fcaronte Dear fcaronte, do you have any idea that the issue will happen again in android14?
> I consider to upgrade to 14 but not sure if the same installer patch can work or there is new patch for android 14.

Nope, have no idea, as I see on my phone I has app working on android 13 and when updating to A14 they continue to work but if I remove, or I need to update or I need to install a new app, then I can't be able to show it without root so for now A14 seem to not work correctly

**Reply by Salafi83** [reply]
> > @fcaronte Dear fcaronte, do you have any idea that the issue will happen again in android14?
> > I consider to upgrade to 14 but not sure if the same installer patch can work or there is new patch for android 14.
> 
> Nope, have no idea, as I see on my phone I has app working on android 13 and when updating to A14 they continue to work but if I remove, or I need to update or I need to install a new app, then I can't be able to show it without root so for now A14 seem to not work correctly 

Have the same exact issue.
Works flawlessly on my second phone with A13.

I updated to A14 on my S23U, everything was working fine until I updated Fermata Auto with AAAD and it stopped showing in AA.
I've tried everything except for downgrading back to A13. 
Even Carstream shows up but just for messages.

**Reply by fcaronte** [reply]
Yes there are app that always show icon because they contain a media control part but if not installed correctly will never work as today no trick was found
